packages/cli/src/handlers/download.ts

```diff
@@ -313,10 +323,10 @@ const getPromoteAction = (action: PromotionAction) => {
         case PromotionAction.NO_CHANGES:
             return 'skipped';
         default:
-            assertUnreachable(action, `Unknown promotion action: ${action}`);
```

Comment:
can we keep this assertUnreachable ? this is done so we make sure we process all the different values in the enum. If we add a new value to the enum, this will fail to compile, forcing us to add a new value.